Automattic/pocket-casts-ios
# Reimagine Sharing: Add sharing app destinations

| 📘 Part of: https://github.com/Automattic/pocket-casts-ios/issues/1910 |
|:---:|

Adds Sharing Destinations to the new sharing modal screen

| Episode | Podcast | Current Position |
| -- | -- | -- |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-06-19 at 22 12 57](https://github.com/Automattic/pocket-casts-ios/assets/3250/b84bf402-4297-43c7-8ffa-ad20f29a8317) | ![Simulator Screenshot - iPhone 15 Pro - 2024-06-19 at 22 13 10](https://github.com/Automattic/pocket-casts-ios/assets/3250/1a9f459e-1fc9-42df-9a7e-6cca74075945) | ![Simulator Screenshot - iPhone 15 Pro - 2024-06-20 at 13 34 43](https://github.com/Automattic/pocket-casts-ios/assets/3250/ddf51ab9-8cd2-4848-89ad-067f4fc0ab21) |

## To test

> [!NOTE]
> Enable the `newSharing` feature flag to test this

### Test Copy Link
* Play a Podcast episode
* Share from the Now Playing Screen
* Choose Episode, Current Position, or Podcast
* Tap "Copy Link" and verify a Toast appears and the link is copied to the clipboard

### Test Share Sheet sharing
* Play a Podcast episode
* Share from the Now Playing Screen
* Choose Episode, Current Position, or Podcast
* Tap "More" and verify a share sheet is presented with a link (same one as Copy Link for now)

## Checklist
